### DISCUSSION TITLE
Positron equivalent to "Use tab for multiline autocompletions" in RStudio

### DISCUSSION THREAD

**Discussion by mattwarkentin** [posit-dev/positron#4333] [Q&A]
Hi,

Is there a Positron setting to enable multiline autocompletions using tabs similar to in RStudio? I haven't found a way to enable this yet.

**Comment by DavisVaughan** [comment]
Where do multiline autocompletions come up?

**Reply by mattwarkentin** [reply]
For example, let's say you have a function with many arguments that you want to span across multiple lines rather than a single line:
```r
# Single line
function(x = 1, y = 2, z = 3)

# Multi line
function(
  x = 1,
  y = 2,
  z = 3
)
```

In the single line example, pressing `TAB` at any point inside the parentheses will show argument autocompletions. However, for the multi-line scenario, pressing `TAB` on an empty line will just add tab/spaces rather than trigger the argument autocompletions. The `Use tab for multiline autocompletions` setting in RStudio enable autocompletions when pressing tab on new lines within function calls.

Hopefully this showcases what I mean by showing the behaviour in RStudio...

https://github.com/user-attachments/assets/3ede73b9-9524-45f2-9422-3f7533d994dd

And the behaviour in Positron...

https://github.com/user-attachments/assets/4b442cf5-a03a-4185-aa9d-350faafc1ca5

**Reply by juliasilge** [reply]
Thanks for this clarification! I've opened https://github.com/posit-dev/positron/issues/4396 to track and prioritize this feature request.
